cmd/gbridge/main.go

```diff
@@ -165,6 +166,38 @@ func main() {
 		Info: proto.DeviceInfo{
 			HwVersion: "1.0",
 		},
+		}); err != nil {
+		log.Fatal(err)
+	}
+
+	// register devices
+	if err := smartHome.RegisterDevice("pborges", gbridge.BasicDevice{
+		Id: "1234567891",
+		Name: proto.DeviceName{
+			Name: "Blind1",
+		},
+		Type: proto.DeviceTypeBlinds,
+		Traits: []gbridge.Trait{
+			gbridge.OpenCloseTrait{
+				DiscreteOnlyOpenClose: true,
+				OpenDirection: []string{"UP","DOWN"},
```

Comment:
since "UP" and "DOWN" are validated by google these should be an enum rather then a string, 

```
type OpenCloseTraitDirection string

const OpenCloseTraitDirectionUp OpenCloseTraitDirection "UP"
```
etc